src/sentry/api/endpoints/event_apple_crash_report.py

```diff
@@ -2,7 +2,13 @@
 
 import six
 
-from django.http import HttpResponse, CompatibleStreamingHttpResponse
+try:
```

Comment:
im a bit concerned about this, but i think its possibly ok due to changes in middleware behavior